app/models/derby_event.rb

```diff
@@ -2,11 +2,21 @@ class DerbyEvent < ActiveRecord::Base
   DATE_FORMAT = '%m/%d/%Y'
   TIME_FORMAT = '%l:%M %P'
 
+  RULESETS = %i(mrda wftda jrda made usars rdcl)
```

Comment:
They might be searchable in the future, but I really just wanted the low effort forms that come with having them separate. Constants are a good idea and I think a nice middle ground between magic strings and a Ruleset model.